.env.local

```diff
@@ -0,0 +1,2 @@
+REACT_APP_BACKEND="https://digicaching.herokuapp.com"
```

Comment:
Just checking, does this file need to be included in the commit?

Reply:
It needs to be included in the heroku server I am not sure how to put it on the dyno otherwise

Reply:
Roger. 👍